content/en/docs/quickstart.md

```diff
@@ -17,7 +17,21 @@ Before you begin, you need:
 Create and run an OCI-compatible registry on your development computer using Docker and the [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry) container image. The following command creates a registry that is accessible at `localhost:5000`.
 
 ```console
-docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.2
+docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.3
+```
+
+> NOTE:
+> Since v1.0.0-rc.1 release, by default, Notation stores signatures using OCI artifact Manifest, which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)). If you choose a different registry, make sure the registry is OCI Image spec v1.1.0 compliant.
+
+### Building OCI-compatible registry image from source
+
+In some cases, such as if your development computer is running an Apple silicon processor, you can download the source of the image for the OCI-compatible registry. The following example clones the repository, builds the image using `docker buildx`, and creates a registry that is available at `localhost:5000`.
```

Comment:
This is suggested by @zr-msft since `docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.3` doesn't work for Apple processor. Maybe we should ask distribution team to rebuild a new image?

Reply:
ideally, an additional image would be great, but building from source is a workaround we can ship near term

@FeynmanZhou this is a known issue: https://docs.docker.com/desktop/troubleshoot/known-issues/